HCIDOCS-651: Oracle Cloud C3 (#88897)
openshift/openshift-docs: modules/c3-assisted-installer-preparing-bastion-server.adoc

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: PROCEDURE
[id="c3-ai-preparing-bastian-server_{context}"]

> = Preparing the {oci} Bastion server

Reviewer: Does "Bastion" need to be capitalized? It looks like it isn't capitalized later in this content.

By implementing a bastion host, you can securely and efficiently manage access to your {oci-c3-no-rt} resources, ensuring that your private instances remain protected and accessible only through a secure, controlled entry point.

.Prerequisites

> * See link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Bastion server - prerequisites].

Reviewer: ```suggestion
* See the Prerequisites section in the  link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```
Because we don't have a way to provide a direct link to each section in the Oracle guide, I revised the link text here and in other places so that customers won't expect, for example, to click the link and immediately see "Prerequisites".

.Procedure

> . Install the bastion server. For details, see link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Bastion Installation].

Reviewer: ```suggestion
. Install the bastion server. For details, see the "Bastion Installation" section of the link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```

> . Install the Terraform application. For details, see link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Terraform installation].

Reviewer: ```suggestion
. Install the Terraform application. For details, see the "Terraform Installation" section of the link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```

> . Install and configure the {oci} command line interface (CLI). For details, see link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Installing and Configuring the {oci} CLI].

Reviewer: ```suggestion
. Install and configure the {oci} command line interface (CLI). For details, see the "Installing and Configuring the {oci} CLI" section in the link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```
